Configuration/TCA/tx_pxaproductmanager_domain_model_attributevalue.php

```diff
@@ -103,21 +103,13 @@
         'attribute' => [
             'label' => 'Attribute',
             'config' => [
-                'type' => 'select',
-                'renderType' => 'selectSingle',
-                'foreign_table' => 'tx_pxaproductmanager_domain_model_attribute',
-                'minitems' => 0,
-                'maxitems' => 1,
+                'type' => 'passthrough',
             ],
         ],
         'product' => [
             'label' => 'product',
             'config' => [
-                'type' => 'select',
-                'renderType' => 'selectSingle',
-                'foreign_table' => 'tx_pxaproductmanager_domain_model_product',
-                'minitems' => 0,
-                'maxitems' => 1,
+                'type' => 'passthrough',
```

Comment:
I think this configuration is useful for interest-based imports as well as reference index. What do you think? The interest extension needs to know if something is a relation, and I think it's not so good to not specify it in the data schema.